Cargo.toml

```diff
@@ -80,6 +80,12 @@ etcetera = { version = "0.10.0" }
 fern = { version = "0.7.0" }
 filetime = { version = "0.2.23" }
 getrandom = { version = "0.3.1" }
+get-size2 = { git = "https://github.com/ibraheemdev/get-size2", rev = "57a38c2", features = [
```

Comment:
This is my personal fork of the `get-size2` crate, which adds some missing features and fixes some inaccuracies. I opened PRs for all of these on the original repository, so we should be able to switch once those are upstreamed.

Reply:
Would it make sense to create a fork in the astral org instead?